src/components/Common/Logo/Logo.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import LogoImg from '../../../assets/Logo.png';
+
+import './Logo.css';
+export interface LogoProps {
+  size?: 'm' | 'l' | 'xl';
+}
+
+const Logo = ({ size = 'l' }: LogoProps) => {
+  return <img className={`logo-${size}`} src={LogoImg} />;
```

Comment:
Also for accessibility purposes it would be nice to add an alt text to the images. If it's just a decorative image you can use an empty string ("").